### ISSUE TITLE
Example code not working (happy and sad both NaN)

### ISSUE THREAD

**Issue by Connum** [EpkCloud/Emotion-Detection-ML-Example#2]
Hi there,

I checked out the code, ran "npm install", then "node index.js", but I get the following output:
```
{ happy: NaN, sad: NaN }
```
Any idea what's going wrong? I'm running on Windows, if that matters.

**Comment by micheleriva** [comment]
Hi @Connum , I'm gonna check it asap 😄

**Comment by momosetti** [comment]
Try to increase iteration number

**Comment by nikkilr88** [comment]
Did anyone figure out why this happens? I'm stuck as well.

**Comment by SpeedyCraftah** [comment]
broken lol

**Comment by ceo0x** [comment]
It also doesnt work even running against "I am sad" which is one of the inputs

**Comment by SpeedyCraftah** [comment]
> It also doesnt work even running against "I am sad" which is one of the inputs

Yeah its not the best, the best approach would be to use nlp.js or use NLU with a public api or something
